src/scene/vcMedia.cpp

```diff
@@ -310,6 +310,14 @@ void vcMedia::HandleSceneExplorerUI(vcState *pProgramState, size_t *pItemID)
   }
 }
 
+void vcMedia::HandleSceneEmbeddedUI(vcState * /*pProgramState*/)
+{
+  const char *imageTypeNames[] = {vcString::Get("scenePOILabelImageTypeStandard"), vcString::Get("scenePOILabelImageTypeOriented"), vcString::Get("scenePOILabelImageTypePanorama"), vcString::Get("scenePOILabelImageTypePhotosphere"), vcString::Get("scenePOILabelImageTypeScreen")};
+  UDCOMPILEASSERT(udLengthOf(imageTypeNames) == vcIT_Count, "Update image names");
+  if (ImGui::Combo(udTempStr("%s##ImageEmbeddedUI", vcString::Get("scenePOILabelImageType")), (int *)&m_image.type, imageTypeNames, (int)udLengthOf(imageTypeNames)))
```

Comment:
What is `m_image.type`? You're blindly casting between pointer types, this is dangerous. At the very least you should add a compile assert that these two types are the same length - if they're not things are going to break. Otherwise, create a local `int` assign to it, use the address of that for the combo, and then assign it back into `m_image.type`, this avoids all type issues:
```c++
int temp = (int)m_image.type; // Correctly casts to int
if (ImGui::Combo(..., &temp, ...))
  ...
m_image.type = (the type...)temp; // Correctly casts back to the right type
```